pages/common/ignite.md

```diff
@@ -0,0 +1,19 @@
+# ignite
+
+> An unfair headstart for your React Native apps.
```

Comment:
We try to refrain from using subjective descriptions in our pages. Please change the language to something more objective and formal.